src/models/user.js

```diff
@@ -1,24 +1,39 @@
+const bcrypt = require("bcrypt");
+
 module.exports = (sequelize, DataTypes) => {
   const User = sequelize.define('User', {
     firstName: {
       type: DataTypes.STRING,
       validate: {
         len: {
           args: [1, 50],
-          msg: 'Please provide field within 1 to 50 characters.'
+          msg: "Please provide 'firstName' field within 1 to 50 characters."
         }
       }
     },
     email: {
       type: DataTypes.STRING,
       allowNull: false,
-      isUnique :true,
+      unique: {
+        args: true,
+        msg: "This Email has been already taken"
+      },
       validate: {
         isEmail: {
           msg: "Email must be Email"
         },
         notEmpty: {
           msg: "Email couldn't be empty"
+        },
+        unique(value, next) {       
+          User.find({
+            where: { email: value },
+            attributes: ['id']
+          }).done((user) => {
+            if (user)
+              return next('This Email has been already taken');
```

Comment:
if you can use one line statement use it
 ```if (user) return next('This Email has been already taken');```
if you want to split it to multi lines, use brackets
```
 if (user) {
              return next('This Email has been already taken');
}
```

Reply:
Yes, it will work anyway, but it's hard to read and later could rise errors if you will be not attentive enough